spacelift/resource_run.go

```diff
@@ -55,16 +67,78 @@ func resourceRun() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"wait": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "Wait for the run to finish",
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:        schema.TypeBool,
```

Comment:
Naive question perhaps but do we need it? Or would the presence of the `wait` block clearly indicate that this is `true`?

Reply:
@marcinwyszynski this gives practitioners the ability to switch waiting on and of by using a variable instead of a dynamic block. Much easier and cleaner to my taste.

Reply:
Yeah that makes sense. But most of the time the intention would be to have it enabled, no? Just wondering then if it could be better UX if the logic was flipped - so the field would be called `disabled`, and `false` by default? I'm slightly worried that if someone forgets to put the `enabled: true` line, the behavior of the resource would fail their expectations.

Reply:
@marcinwyszynski Fair objection. Changed it according you proposal